common/views/components/IIIFViewerPrototype/IIIFViewerPrototype.tsx

```diff
@@ -73,87 +73,115 @@ const ZoomedImagePrototype = dynamic(
   }
 );
 
-const ImageViewerControls = styled.div<{ showControls?: boolean }>`
-  position: fixed;
-  bottom: 0;
-  left: 73%;
-  z-index: 1;
-  opacity: ${props => (props.showControls ? 1 : 0)};
-  transition: opacity 300ms ease;
-  display: flex;
-  /* TODO: keep an eye on https://github.com/openseadragon/openseadragon/issues/1586
-    for a less heavy handed solution to Openseadragon breaking on touch events */
-  &,
-  button,
-  a {
-    touch-action: none;
-  }
+const Grid = styled.div`
+  display: grid;
+  height: calc(100vh - 85px); // FIXME: use variable for header height
+  overflow: hidden;
+  grid-template-columns: [left-edge] minmax(200px, 3fr) [desktop-sidebar-end main-start desktop-topbar-start] 9fr [right-edge];
+  grid-template-rows: [top-edge] min-content [desktop-main-start desktop-topbar-end] 1fr [mobile-bottombar-start mobile-main-end] min-content [bottom-edge];
 
-  button {
-    display: block;
-  }
+  .viewer-desktop {
+    display: none;
 
-  .icon {
-    margin: 0;
+    ${props => props.theme.media.medium`
+      display: inherit;
+    `}
   }
 
-  .btn__text {
-    border: 0;
-    clip: rect(0 0 0 0);
-    height: 1px;
-    margin: -1px;
-    overflow: hidden;
-    padding: 0;
-    position: absolute;
-    width: 1px;
-    white-space: nowrap;
+  .viewer-mobile {
+    ${props => props.theme.media.medium`
+      display: none;
+    `}
   }
-`;
 
-const Grid = styled.div`
-  display: grid;
-  height: calc(100vh - 85px); // FIXME: use variable for header height
-  overflow: hidden;
-  grid-template-columns: [left-edge] minmax(200px, 330px) [desktop-sidebar-end main-start desktop-topbar-start] 9fr [right-edge];
-  grid-template-rows: [top-edge] min-content [desktop-main-start desktop-topbar-end] 1fr [bottom-edge];
+  ${props => props.theme.media.xlarge`
+    grid-template-columns: [left-edge] minmax(200px, 330px) [desktop-sidebar-end main-start desktop-topbar-start] 9fr [right-edge];
+  `}
 `;
 
-const Sidebar = styled.div<{ isActive: boolean }>`
-  display: ${props => (props.isActive ? 'inherit' : 'none')};
-  grid-area: top-edge / left-edge / bottom-edge / desktop-sidebar-end;
+const Sidebar = styled.div<{
+  isActiveMobile: boolean;
+  isActiveDesktop: boolean;
+}>`
+
+  display: ${props => (props.isActiveMobile ? 'inherit' : 'none')};
+
+  ${props => props.theme.media.medium`
+    display: ${props => (props.isActiveDesktop ? 'inherit' : 'none')};
+  `}
+
+  grid-area: top-edge / left-edge / bottom-edge /right-edge;
+
+  ${props => props.theme.media.medium`
+    grid-area: top-edge / left-edge / bottom-edge / desktop-sidebar-end;
```

Comment:
What are we saying for support of grids? IE-11 still needs a prefix, and I can't remember if we're post-applying vendor prefixes, especially in styled components?